site/content/en/docs/manual/basics/3d-object-annotation.md

```diff
@@ -0,0 +1,109 @@
+---
+title: '3D Object annotation'
+linkTitle: '3D Object annotation'
+weight: 19
+description: 'Overview of basic operations available when annotating 3D objects.'
+---
+
+Use the 3D Annotation tool for labeling 3D objects and scenes, such as vehicles, buildings, landscapes, and others.
+
+See:
+
+- [Navigation](#navigation)
+- [Annotation with cuboids](#annotation-with-cuboids)
+  - [Annotation with shapes](#annotation-with-shapes)
+  - [Tracking with cuboids](#tracking-with-cuboids)
+
+## Navigation
+
+The 3D annotation canvas looks like the following
+
+![3D canvas](/images/3d-canvas.jpg)
+
+> Note: if you added contextual images to the dataset, the canvas will include them. For more information, see [Contextual images](docs/manual/advanced/context-images/)
+
+For information on the available tools, see [Controls sidebar](http://localhost:1313/docs/manual/basics/controls-sidebar/).
+
+You can navigate, using the mouse, or navigation keys:
+
+![](/images/image216_carla_town3.jpg)
+
+You can also use keyboard shortcuts to navigate:
+
+<!--lint disable maximum-line-length-->
+
+| Action          | Keys                                          |
+| --------------- | --------------------------------------------- |
+| Camera rotation | **Shift** + **Arrow** (Up, Down, Left, Right) |
+| Left/Right      | **Alt**+**J**/ **Alt**+**L**                  |
+| Up/down         | **Al**t+**U**/ **Alt**+**O**                  |
+| Zoom in/ou      | **Alt**+**K**/ **Alt**+**I**                  |
+
+<!--lint enable maximum-line-length-->
+
+## Annotation with cuboids
+
+There are two options available for 3D annotation:
+
+- **Shape**: for tasks like object detection, segmentation, and classification.
```

Comment:
detection - fine
segmentation is not supported for 3D
for classification we use tags, they also not supported on 3D scene

Reply:
updated in the doc